### ISSUE TITLE
Build Looking Glass module for ARM chip (like Raspberry Pi)

### ISSUE THREAD

**Issue by dev-sora** [Kitware/LookingGlassVTKModule#66]
Thanks Kitware team,

I'd like to build VTK with enabling Looking Glass module for Raspberry Pi, but the shared library is compiled for x86_64 which preventing me from natively building on the board.
Is there any good practice to perform native build for arm chipset?

Thanks

**Comment by psavery** [comment]
Hi there,

It's been a while since I worked on this. We do have python shared libraries available for macos arm64 on PyPI, but not for Linux.

You should be able to compile this for arm on Linux, though. There are details about how to compile it in the [README](https://github.com/Kitware/LookingGlassVTKModule?tab=readme-ov-file#looking-glass-device-support). I can't remember enough details to provide any further info, though.

Thank you!
Patrick

**Comment by aylward** [comment]
Patrick, do they perhaps need to contact looking glass to get the appropriate sdk / library for their system?On Jun 14, 2024 12:38 PM, Patrick Avery ***@***.***> wrote:
Hi there,
It's been a while since I worked on this. We do have python shared libraries available for macos arm64 on PyPI, but not for Linux.
You should be able to compile this for arm on Linux, though. There are details about how to compile it in the README. I can't remember enough details to provide any further info, though.
Thank you!
Patrick

**Comment by psavery** [comment]
Yes that's true. You will probably have to contact Looking Glass to get the library for arm64 Linux.
